apps/src/studio/studio.js

```diff
@@ -789,8 +850,12 @@ function spriteCollisionDistance  (i1, i2, yAxis) {
 }
 
 function spriteCollidableCollisionDistance (iS, collidable, yAxis) {
```

Comment:
Realize this is existing code, but while you're in here could you add @param comments for these? `iS` seems particularly confusing to me initially. Presumably this is meant to be spriteIndex?